src/main/java/com/passpnu/passwordmanager/service/ServiceEntityServiceImpl.java

```diff
@@ -1,7 +1,95 @@
 package com.passpnu.passwordmanager.service;
 
+import com.passpnu.passwordmanager.dto.ServiceDto;
+import com.passpnu.passwordmanager.entity.ServiceEntity;
+import com.passpnu.passwordmanager.repos.ServiceRepository;
+import lombok.AllArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import javax.naming.NameNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
 @Service
-public class ServiceEntityServiceImpl {
+@AllArgsConstructor
+public class ServiceEntityServiceImpl implements ServiceEntityService{
+    ServiceRepository serviceRepository;
+
+    private ServiceEntity dtoToEntity(ServiceDto serviceDto){
```

Comment:
This is not an expected functionality of service. You should create special code-units (mappers) for that, better with Mapstruct

Reply:
Mapper is added